presto-accumulo/src/main/java/com/facebook/presto/accumulo/io/AccumuloRecordCursor.java

```diff
@@ -131,6 +129,9 @@ public AccumuloRecordCursor(
             }
         }
 
+        IteratorSetting setting = new IteratorSetting(21, WholeRowIterator.class);
```

Comment:
Accumulo has an iterator stack that runs on the server side.  Higher the priority, the later it runs in the pipeline.  This iterator should run last, and the default visibility iterator (which runs on every scan) has a priority of 20 -- which brings this to 21.  It could alternatively be `Integer.MAX_VALUE` if that would be more clear (along with a comment on what this means).